content/sensu-go/5.6/reference/filters.md

```diff
@@ -343,54 +343,44 @@ example      | {{< highlight shell >}} "annotations": {
 
 ### Handling production events
 
-The following example filter definition, entitled `production_filter`, will
-match event data with a custom entity definition attribute `"namespace":
-"production"`.
+The following filter,`production_filter`, only handles events with a custom entity label `"environment": "production"`.
 
 {{< highlight json >}}
 {
   "type": "EventFilter",
   "api_version": "core/v2",
   "metadata": {
     "name": "production_filter",
-    "namespace": "default",
-    "labels": null,
-    "annotations": null
+    "namespace": "default"
   },
   "spec": {
     "action": "allow",
     "expressions": [
-      "event.entity.namespace == 'production'"
-    ],
-    "runtime_assets": []
+      "event.entity.labels.environment == 'production'"
+    ]
   }
 }
 {{< /highlight >}}
 
 ### Handling non-production events
 
-The following example filter definition, entitled `development_filter`, will
-discard event data with a custom entity definition attribute `"namespace":
-"production"`.
-
+The following filter,`development_filter`, discards events with a custom entity label `"environment": "production"`.
```

Comment:
Is the intention of this filter to let events be handled when there is no label at all named `environment`? I tested with an entity that had no labels and events were still handled, as the expression is coming back as false.

Also, the naming is a little misleading as `development_filter`. It led me to believe that the label had to be `development`. In it's current state it is just "not production" and I'll handle you.

Reply:
The intention is to allow only events that do not have the `environment: production` label, so events without an `environment` label and events with `environment` set to something other than `production` would be allowed.

+1 to the name change

Reply:
Sounds good to me! 
```suggestion
The following filter,`not_production`, discards events with a custom entity label `"environment": "production"`.
```